BioSQL/BioSeq.py

```diff
@@ -130,10 +132,18 @@ def toseq(self):
         return Seq(str(self), self.alphabet)
 
     def __add__(self, other):
+        """Add another sequence or string to this sequence.
+
+        The sequence is first converted to a Bio.Seq object before the addition.
```

Comment:
I think it is unclear from text which object gets turned into a Seq object. How about something like ``This sequence is first converted to a Seq object before the addition of the other sequence.``? Should we say it will return a ``Seq`` object (not a ``DBSeq`` object)?